packaging/common/rabbitmq-server-ha.ocf

```diff
@@ -0,0 +1,1585 @@
+    local make_amnesia=false
+    local rc=$OCF_ERR_GENERIC
+
+    # check status of a beam process
+    get_status
+    rc=$?
+    if [[ $rc == 0 ]] ; then
+        # beam is running
+        # check status of rabbit app and stop it, if it is running
+        get_status rabbit
+        rc=$?
+        if [[ $rc == 0 ]] ; then
+            # rabbit app is running, have to stop it
+            ocf_log info "${LH} Stopping RMQ-app prior to reset the mnesia."
+            stop_rmq_server_app
+            rc=$?
+            if [[ $rc != 0 ]] ; then
+                 ocf_log warn "${LH} RMQ-app can't be stopped."
+                 make_amnesia=true
+            fi
+        fi
+
+        if ! $make_amnesia ; then
+            # rabbit app is not running, reset mnesia
+            ocf_log info "${LH} Execute reset with timeout: ${TIMEOUT_ARG}"
+            su_rabbit_cmd "${OCF_RESKEY_ctl} reset"
+            rc=$?
+            if [[ $rc != 0 ]] ; then
+                ocf_log info "${LH} Execute force_reset with timeout: ${TIMEOUT_ARG}"
+                su_rabbit_cmd "${OCF_RESKEY_ctl} force_reset"
+                rc=$?
+                if [[ $rc != 0 ]] ; then
+                    ocf_log warn "${LH} Mnesia couldn't cleaned, even by force-reset command."
+                    make_amnesia=true
+                fi
+            fi
+        fi
+    else
+        # there is no beam running
+        make_amnesia=true
+        ocf_log warn "${LH} There is no Beam process running."
+    fi
+
+    # remove mnesia files, if required
+    if $make_amnesia ; then
+        kill_rmq_and_remove_pid
+        ocf_run rm -rf ${MNESIA_FILES}*
+        ocf_log warn "${LH} Beam have been killed. Mnesia files appear corrupted and have been removed."
+    fi
+    # always return OCF SUCCESS
+    return $OCF_SUCCESS
+}
+
+
+block_client_access()
+{
+    # do not add temporary RMQ blocking rule, if it is already exist
+    # otherwise, try to add a blocking rule with max of 5 retries
+    local tries=5
+    until $(iptables -nvL | grep -q 'temporary RMQ block') || [[ $tries -eq 0 ]]; do
+      ((tries--))
+      iptables -I INPUT -p tcp -m tcp --dport ${OCF_RESKEY_node_port} -m state --state NEW,RELATED,ESTABLISHED \
+      -m comment --comment 'temporary RMQ block' -j REJECT --reject-with tcp-reset
+      sleep 1
+    done
+    if [ $tries -eq 0 ]; then
+        return $OCF_ERR_GENERIC
+    else
+        return $OCF_SUCCESS
+    fi
+}
+
+unblock_client_access()
+{
+    # remove all temporary RMQ blocking rules, if there are more than one exist
+    for i in $(iptables -nvL --line-numbers | awk '/temporary RMQ block/ {print $1}'); do
+      iptables -D INPUT -p tcp -m tcp --dport ${OCF_RESKEY_node_port} -m state --state NEW,RELATED,ESTABLISHED \
+      -m comment --comment 'temporary RMQ block' -j REJECT --reject-with tcp-reset
+    done
+}
+
+get_nodes__base(){
+    local infotype=''
+    local rc=$OCF_ERR_GENERIC
+
+    if [ "$1" == 'nodes' ]
```

Comment:
The syntax of this test is invalid: `[` doesn't support `==` as an operator, but see next comment.